README.md

```diff
@@ -51,16 +50,22 @@ See also the [examples](examples).
 ### Server
 
 The `Server` class is responsible for handling incoming connections and then
-emit a `request` event for each incoming HTTP request.
+execute the execute the callback function passed to the constructor.
 
 It attaches itself to an instance of `React\Socket\ServerInterface` which
 emits underlying streaming connections in order to then parse incoming data
-as HTTP:
+as HTTP.
+
+For each incoming connection, it executes the callback function with the
+respective [`Request`](#request) and [`Response`](#response) objects:
```

Comment:
Which callback function? (see above) How about this:

For each incoming **request, it executes the callback function passed to the constructor** with…